pyconcz/templates/default/pages/minimal_landing.html

```diff
@@ -14,7 +14,7 @@ <h1>PyCon CZ 2019</h1>
                 <div>Ostrava</div>
                 <div>June 14–16</div>
             </div>
-            <div class="pc-image-copyright">Photo © <a href="https://www.zerzon.cz/">Jiří Zerzoň</a></div>
+            <div class="pc-image-copyright">Photo 🄯 <a href="https://www.flickr.com/photos/kubajs/20958638705/">Jakub Souček</a></div>
```

Comment:
What does this square character mean?

Reply:
I see copyleft symbol https://www.fileformat.info/info/unicode/char/1f12f/index.htm

Reply:
U+1F12F: COPYLEFT SYMBOL

But the license is CC BY 2.0, not copyleft. I'd just say "by".

Reply:
right.

Reply:
the CC unicode symbol is probably still not approved :(